src/lib/sql/sql_translator.cpp

```diff
@@ -1370,6 +1370,10 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_create_table(const hs
           std::cout << "WARNING: Parsing TIME to string since date and time data types are not yet supported\n";
           column_definition.data_type = DataType::String;
           break;
+        case hsql::DataType::BOOLEAN:
+          std::cout << "WARNING: Parsing BOOLEAN to string\n";
```

Comment:
Add information how this looks like? "TRUE", "true", "1"?

Reply:
This depends on what the user adds to that table. We do not parse the input values, so it could be any string the user inserts. Postgres [supports a variety of options](https://www.postgresql.org/docs/14/datatype-boolean.html), but we do not enforce a date format (unless if it's a date literal or cast as date) either.